javascript-constructors/example-constructor.js

```diff
@@ -0,0 +1,9 @@
+function ExampleConstructor() {
+  console.log(ExampleConstructor.prototype);
```

Comment:
you don't want anything in the constructor, these console logs should be outside.